src/webgpu/api/operation/rendering/depth_bias.spec.ts

```diff
@@ -99,10 +100,11 @@ class DepthBiasTest extends GPUTest {
 
     const depthStencilAttachment: GPURenderPassDepthStencilAttachment = {
       view: depthTexture.createView(),
-      depthLoadOp: 'load',
+      depthLoadOp: depthStencilLoadOp,
       depthStoreOp: 'store',
-      stencilLoadOp: 'load',
+      stencilLoadOp: depthStencilLoadOp,
       stencilStoreOp: 'store',
+      depthClearValue: depthStencilLoadOp === 'clear' ? 0.4 : undefined,
```

Comment:
depthStencilLoadOp is a roundabout way of choosing an initial depth (either 0.0 or 0.4). Pushed a commit to make it more straightforward